src/utils/DisabledDate.js

```diff
@@ -0,0 +1,243 @@
+/* eslint-disable no-underscore-dangle */
+
+import makeCellUtils from './cellUtils'
+
+export default class DisabledDate {
+  constructor(utils, disabledDates) {
+    this._utils = utils
+    this._disabledDates = disabledDates
+  }
+
+  get config() {
+    const dd = this._disabledDates
```

Comment:
Do you mind rename such short variables? At first i was pretty confused what `dd` and `u` was about. I needed the declaration line to understand it.
A variable should always be understandable by itself. This will make it easier to read the code.

In this example it still can be `disabledDates` and `utils`

Reply:
Fair comment... will do.